constant.py

```diff
@@ -23,8 +23,12 @@
 HTTP_PROTOCOL = "http://"
 API_VERSION = "/v1"
 
+GRAFANA_API_VERSION = "/v1"
+GRAFANA_API = "/api"
+GRAFANA_PROVISIONING = "/provisioning"
+
 # TODO add command parameter to set this DEV
-DEV = False
+DEV = True
```

Comment:
Should be left to false

Reply:
fixed, I'll always miss this one I guess